src/App.styled.ts

```diff
@@ -35,111 +35,119 @@ const List = styled(BaseLayoutElement)`
 `;
 
 export const Info = styled.div`
-	grid-column: 1 / 2;
-	grid-row: 1 / 5;
 	display: flex;
 	flex-direction: column;
 	justify-content: center;
+	grid-column: 1;
+	grid-row: 1;
 
-	@media (max-width: 768px) {
-		grid-column: 1;
-		grid-row: 1;
+	@media (min-width: 768px) {
+		grid-column: 1 / 2;
+		grid-row: 1 / 5;
 	}
 `;
 
 export const Editor = styled(BaseLayoutElement)`
 	padding: 0;
-	grid-column: 1 / 2;
-	grid-row: 5 / 31;
-
-	@media (max-width: 768px) {
-		grid-column: 1;
-		grid-row: 2;
-		height: 50vh;
+	grid-column: 1;
+	grid-row: 2;
+	height: 50vh;
+
+	@media (min-width: 768px) {
+		grid-column: 1 / 2;
+		grid-row: 5 / 31;
+		height: 100%;
 	}
 `;
 
 export const WebApi = styled(List)`
-	grid-column: 1 / 2;
-	grid-row: 31 / 37;
-
-	@media (max-width: 768px) {
-		grid-column: 1;
-		grid-row: 3;
-		height: 10vh;
+	grid-column: 1;
+	grid-row: 3;
+	height: 10vh;
+
+	@media (min-width: 768px) {
+		grid-column: 1 / 2;
+		grid-row: 31 / 37;
+		height: 85%;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
❔ question:
May I ask why do we need to specify height if it's already defined by height of the viewport and amount grid-row directive? 
What do you think about keeping it unset to override `10vh`?
Applicable for all other places in the PR...
```suggestion
		height: unset;
```